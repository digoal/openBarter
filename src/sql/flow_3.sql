set search_path = ob;
select ob_fadd_account('own1','q1',100);
select ob_fadd_account('own1','q1',200);
select ob_fsub_account('own1','q1',300);
select ob_fget_errs();
select count(*) from ob_tstock; -- no stock remains
truncate ob_tquality cascade;
