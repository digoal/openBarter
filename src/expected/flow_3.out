set search_path = ob;
select ob_fadd_account('own1','q1',100);
 ob_fadd_account 
-----------------
               0
(1 row)

select ob_fadd_account('own1','q1',200);
 ob_fadd_account 
-----------------
               0
(1 row)

select ob_fsub_account('own1','q1',300);
 ob_fsub_account 
-----------------
               0
(1 row)

select ob_fget_errs();
 ob_fget_errs 
--------------
            0
(1 row)

select count(*) from ob_tstock; -- no stock remains
 count 
-------
     0
(1 row)

truncate ob_tquality cascade;
NOTICE:  truncate cascades to table "ob_tstock"
NOTICE:  truncate cascades to table "ob_tnoeud"
NOTICE:  truncate cascades to table "ob_tmvt"
NOTICE:  truncate cascades to table "ob_tcommit"
